packages/angular/cli/src/analytics/analytics.ts

```diff
@@ -220,7 +167,6 @@ export async function hasGlobalAnalyticsConfiguration(): Promise<boolean> {
  */
 export async function getGlobalAnalytics(): Promise<AnalyticsCollector | undefined> {
   analyticsDebug('getGlobalAnalytics');
-  const propertyId = AnalyticsProperties.AngularCliDefault;
 
   if ('NG_CLI_ANALYTICS' in process.env) {
     if (process.env['NG_CLI_ANALYTICS'] == 'false' || process.env['NG_CLI_ANALYTICS'] == '') {
```

Comment:
We we expect users to set this env variable to determine that we are running in CI. Why not use `CI` instead?

Reply:
I agree that `CI` would be more consistent. No one is going to go out of their way to set `NG_CLI_ANALYTICS=ci`, so I feel like this value is pretty meaningless as is. We should definitely fall back to `CI` if `NG_CLI_ANALYTICS` is not set.

Reply:
SGTM